BubbleSortTest.java

```diff
@@ -0,0 +1,43 @@
+package SortingAlorithms;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Random;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BubbleSortTest {
```

Comment:
bubble sort test i select sort test to powinna byc jedna klasa przyjmujaca metode sortujaca, ktora ma przetestowac. l. rozgrzewkowa wiec ok